### PR TITLE
[Bexley] use Symology Event Type as external status

### PR DESCRIPTION
**Pull request by struan**
Use the Event Type column as the external status code if present in the
update CSV.

Fixes mysociety/fixmystreet-commercial#2279

### CODE REVIEW

File: perllib/Open311/Endpoint/Integration/UK/Bexley/Symology.pm [hunk 1]
```diff
@@ -71,6 +71,12 @@ sub _row_status {
     };
 }
 
+sub _row_external_status_code {
+    my ($self, $row, $status) = @_;
+    return undef unless $status eq 'not_councils_responsibility';
```
Comment (davea) [inline perllib/Open311/Endpoint/Integration/UK/Bexley/Symology.pm:76]: `$status` may be `undef` here which causes an uninitialized value warning, how about:

```suggestion
    return undef unless $status && $status eq 'not_councils_responsibility';
```

**Review by davea — CHANGES_REQUESTED** [review]
One warning to squash is all.

**Review by davea — APPROVED** [review]
LGTM!

**Comment by struan** [comment]
Extra mapping of categories added

**Review by chrismytton — APPROVED** [review]
🟢

**Pull request merged**